components/bitly/common/common.utils.mjs

```diff
@@ -0,0 +1,72 @@
+const formatQueryString = (payload) => {
```

Comment:
I don't get exactly the idea from here. Are you trying to format the payload to use in the `query`?  if so, axios has a param called `params` where you can send query params. It will be automatically